packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -227,24 +227,45 @@ export const RemoteServerFactory = {
      */
 
     _setupLogging(app) {
+        const loggerFormat = function (tokens, req, res) {
+            const contentLength = tokens.res(req, res, 'content-length')
+            return [
+                `(${res.locals.requestId})`,
+                tokens.method(req, res),
+                tokens.url(req, res),
+                tokens.status(req, res),
+                tokens['response-time'](req, res),
+                'ms',
+                contentLength && `- ${contentLength}`
+            ].join(' ')
+        }
+
+        // Morgan stream for logging status codes less than 400
```

Comment:
I believe that we have removed Morgan, so this comment should be removed or updated.

Reply:
We haven't removed Morgan. We just removed the usage of `morgan-streaming.js` because it is unnecessary. 
Morgan is still the express middleware we use to log incoming requests. 
I think is useful to have the comment to understand the two different streams we use depending on the status code.